src/commonMain/kotlin/exchange.dydx.abacus/state/manager/AsyncAbacusStateManagerProtocol.kt

```diff
@@ -111,6 +111,7 @@ interface AsyncAbacusStateManagerSingletonProtocol {
     var subaccountNumber: Int
     var market: String?
     var cosmosWalletConnected: Boolean?
+    var solanaWalletConnected: Boolean?
```

Comment:
Can this be refactored as an enum, since the wallet connection types are mutually exclusive?

```
enum class WalletConnectionType {
    Ethereum,
    Cosmos,
    Solana
}

...
var walletConnectionType: WalletConnectionType?
```

Reply:
Done: https://github.com/dydxprotocol/v4-abacus/pull/656